### PR TITLE
Update enable-features-office-365.md

### PR DESCRIPTION
**Pull request by Exnav29**
Added the "Network planner" setting.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/enable-features-office-365.md [hunk 1]
```diff
@@ -99,6 +99,10 @@ You can specify a coexistence mode: **Teams only**, **Islands** (Teams and Skype
 
 Here you can choose the app that users will use to join Skype for Business meetings (Skype for Business or the [Skype Meetings App](https://support.office.com/en-us/article/What-is-Skype-Meetings-App-Skype-for-Business-Web-App-1FF3D412-718A-4982-8FF2-A4992608CDB5)). This setting isn't dependent on the coexistence mode setting.
 
+#### Network planner
```
Comment (nenonix) [inline Teams/enable-features-office-365.md]:
```suggestion
#### Network Planner
```

File: Teams/enable-features-office-365.md [hunk 2]
```diff
@@ -99,6 +99,10 @@ You can specify a coexistence mode: **Teams only**, **Islands** (Teams and Skype
...
+
+Network planner helps you to determine and organize network requirements for connecting people that use Teams across your organization in a few steps. By providing your networking details and Teams usage, you get calculations and the network requirements you need when deploying Teams and cloud voice across organizational physical locations.
```
Comment (nenonix) [inline Teams/enable-features-office-365.md]:
```suggestion
Network Planner helps you determine and organize network requirements for connecting Teams users across your organization.  Learn how to [Use the Network Planner for Microsoft Teams](https://docs.microsoft.com/microsoftteams/network-planner).
```

**Comment by scanum** [comment]
@Exnav29 Could you please make changes? Thanks!

**Comment by Exnav29** [comment]
@scanum I think I made the changes please let me know if I missed something.

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**